src/failover.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const MIN_BACKOFF_DURATION = 30_000; // 30 seconds in milliseconds
+const MAX_BACKOFF_DURATION = 10 * 60 * 1000; // 10 minutes in milliseconds
+const JITTER_RATIO = 0.25;
+
+export function getFixedBackoffDuration(timeElapsed: number): number | undefined {
+    if (timeElapsed <= 100_000) { // 100 seconds in milliseconds
```

Comment:
.NET uses '<' comparison instead of '<='

https://github.com/Azure/AppConfiguration-DotnetProvider/blob/65ed48033482e796143b41fa150d06293fbf8316/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs#L120-L128